python/src/aiconfig/editor/client/src/components/prompt/PromptContainer.tsx

```diff
@@ -38,6 +38,25 @@ type Props = {
 
 export const PROMPT_CONTAINER_HEIGHT_MAP = new Map<string, number>();
 
+// Set default styles, which can be overridden by the provided themeOverride
+// or theme associated with mode
+const useStyles = createStyles((theme) => ({
+  cellStyle: {
+    borderBottomRightRadius: "0px",
+    borderTopRightRadius: "0px",
+    flex: 1,
+  },
+
+  sidePanel: {
+    border: `1px solid ${
+      theme.colorScheme === "dark" ? theme.colors.dark[4] : theme.colors.gray[3]
```

Comment:
This is the main 'new' style. Basically, just matching up the border colour to what the cell is using for the mantine Card. And that's defined here:
https://github.com/mantinedev/mantine/blob/7b1457f9eb084f0fb663c25793d08877cc957db0/packages/%40mantine/core/src/components/Card/Card.module.css#L43-L49